#msg pour le topic du regulateur

float64 pos0
float64 pos1
float64 cap
float64 obja0
float64 obja1
float64 objb0
float64 objb1 
float64 deltaMax